src/libsyntax/config.rs

```diff
@@ -242,7 +234,17 @@ impl<T: CfgFolder> fold::Folder for T {
     }
 
     fn fold_item(&mut self, item: P<ast::Item>) -> SmallVector<P<ast::Item>> {
-        self.configure(item).map(|item| SmallVector::one(item.map(|i| self.fold_item_simple(i))))
+        self.configure(item).map(|item| fold::noop_fold_item(item, self))
```

Comment:
This is a pure uninlining refactoring -- c.f. [`noop_fold_item`](https://github.com/rust-lang/rust/blob/master/src/libsyntax/fold.rs#L1026-L1029).